std_msgs/Header header

float64 steering_angle_cmd
float64 steering_wheel_angle_cmd
int steering_wheel_percentage

float64 accel_cmd
float64 decel_cmd
float64 speed_cmd

geometry_msgs/Twist twist

int gear_cmd

bool enable
bool disable
bool autonomous_mode




